bundler.d/test.rb

```diff
@@ -14,7 +14,7 @@
   gem 'selenium-webdriver', :require => false
   gem 'shoulda-matchers', '>= 4.0', '< 4.4'
   gem 'shoulda-context', '~> 1.2'
-  gem 'as_deprecation_tracker', '~> 1.4'
+  gem 'as_deprecation_tracker', :git => 'https://github.com/evgeni/as_deprecation_tracker', :branch => 'ruby3'
```

Comment:
Version 1.6.0 has your patches.
```suggestion
  gem 'as_deprecation_tracker', '~> 1.6'
```

Reply:
Merged in 68eb570221c0737d4eaa547eeebf1a550057ee49, which now gives a merge conflict.